### PR TITLE
Implement C415

### PR DESCRIPTION
**Pull request by harupy**
#305

### CODE REVIEW

File: src/ast/checks.rs [hunk 1]
```diff
@@ -985,6 +985,44 @@ pub fn unnecessary_collection_call(
...
+                                    op: Unaryop::USub,
+                                    operand,
+                                } = &step.node
+                                {
+                                    if let ExprKind::Constant {
+                                        value: Constant::Int(val),
+                                        ..
+                                    } = &operand.node
+                                    {
+                                        if val.to_string() == "1" {
```
Comment (harupy) [inline src/ast/checks.rs:1006]: Tried `val == 1` first, but got this error:

image

Comment (charliermarsh) [reply src/ast/checks.rs:1006]: Would it be safer to create BigInt(1) and compare them as BigInts?

Comment (harupy) [reply src/ast/checks.rs]: Yeah, that would be safer. Fixed :)

**Comment by charliermarsh** [comment]
Nice work :)

**Pull request merged**